other/bootstrap_daemon/tox_bootstrap.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /var/run/tox-bootstrapd.pid
```

Comment:
PID would just be optional. Systemd doesn't require it to stop a process, so there is no need to use it with systemd. Sysvinit requires, so it could be used there. Someone might want to just launch it directly, without any init system, in which case PID wouldn't be that much necessary.